Psychtoolbox/PsychCal/MeasureDpi.m

```diff
@@ -55,9 +81,22 @@
     [window,screenRect]=Screen('OpenWindow',theScreen);
     white=WhiteIndex(window);
     black=BlackIndex(window);
-
+    
+    %This might work if you have a modern monitor with reasonable updated
+    %drivers and at least windows 10.
```

Comment:
Why is it dependent on Windows 10? It should also work on Windows < 10, macOS and Linux. Any reason it wouldn't?

Reply:
I assume that the script [Screen.mex] makes use of _GetMonitorDisplayAreaSize_ function or similar, which requires Windows Vista or superior. Additionally, as far as I know, the EDID connection between monitors and the OS was improved in W10. 
From my personal experience, a laptop with W7 was unable to retrieve the physical size properly, but after updating it to W10 it worked.
Regarding the other OS, unfortunately, I don't know and I can not currently test it.
Nonetheless, it might be worthy to disclaim that it won't work with projectors or old CRT monitors.
 FYI : [Stackoverflow discussion on the topic](https://stackoverflow.com/questions/12652835/getting-actual-screen-dpi-ppi-under-windows)